### PR TITLE
feat(remix): Add `meta` instructions.

### PR DESCRIPTION
**Pull request by onurtemizkan**
Adds `meta` function instructions for Distributed Tracing back. 

Also, includes changes in the open PR https://github.com/getsentry/sentry-javascript/pull/9493

Needs updates depending on that.

### CODE REVIEW

**Comment by shanamatthews** [comment]
Hey @onurtemizkan - tag me when this is ready for review and I'll make sure it get's reviewed for ya!

**Comment by onurtemizkan** [comment]
Hey @shanamatthews, https://github.com/getsentry/sentry-javascript/pull/9493 is merged and this is now ready for reviews!

File: src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx [hunk 1]
```diff
@@ -1,4 +1,37 @@
-If you're using the current version of our Remix SDK, distributed tracing will work out of the box for the client and server. To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, you should define `tracePropagationTargets` for client-side.
+Remix SDK attaches `sentry-trace` and `baggage` values from your `root` loader. You need to use [`meta`](https://remix.run/docs/en/v1/api/conventions#meta) function to attach the data from your `loader` as `<meta>` tags. The following code snippet shows how to do this:
```
Comment (shanamatthews) [inline src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx:1]:
```suggestion
Remix SDK attaches `sentry-trace` and `baggage` values from your `root` loader. You need to use the [`meta`](https://remix.run/docs/en/v1/api/conventions#meta) function to attach the data from your `loader` as `<meta>` tags. The following code snippet shows how to do this:
```

File: src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+    // ...
+    "sentry-trace": data.sentryTrace,
+    baggage: data.sentryBaggage,
+  };
+};
+```
+
+```typescript {tabTitle: Remix v2} {filename: root.tsx}
+// Sentry provides a type for Remix v2 MetaFunction parameters,
+// so you can access `data.sentryTrace` and `data.sentryBaggage` alongside other data from loader.
```
Comment (shanamatthews) [inline src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx:15]:
```suggestion
// Sentry provides a type for Remix v2 MetaFunction parameters
// so you can access `data.sentryTrace` and `data.sentryBaggage` alongside other data from loader.
```

File: src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx [hunk 3]
```diff
@@ -1,4 +1,37 @@
...
+      name: "baggage",
+      content: data.sentryBaggage,
+    },
+  ];
+};
+```
+
+The `name` attributes must be the strings `"sentry-trace"` and `"baggage"` and the `content` attributes must be generated by your backend Sentry SDK. For `sentry-trace`, use `span.toSentryTrace()` (or equivalent, depending on the backend platform). This guarantees that a new and unique value will be generated for each request. For `baggage`, use `serializeBaggage(span.getBaggage())`.
+
+To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, you should define `tracePropagationTargets` for client-side.
```
Comment (shanamatthews) [inline src/platform-includes/distributed-tracing/how-to-use/javascript.remix.mdx:34]:
```suggestion
To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, you should define `tracePropagationTargets` on the client side. See [Dealing with CORS Issues](/usage/distributed-tracing/dealing-with-cors-issues/) for more information.
```

**Review by shanamatthews — APPROVED** [review]
Good from me. @AbhiPrasad - could you review this for accuracy too?

**Pull request merged**